JobConfig/primary/flatproton.fcl

```diff
@@ -3,41 +3,23 @@
 #
 # original author: Dave Brown, LBNL
 #
-#include "JobConfig/primary/prolog.fcl"
-process_name: proton
-
-source: { module_type : EmptyEvent }
-services: @local::Services.Sim
-physics : { 
-  producers : {
-    generate : {
+#include "JobConfig/primary/primary.fcl"
+# set the generator
+physics.producers.generate : {
       module_type: StoppedParticleReactionGun
       verbosityLevel : 1
       muonStops: @local::mu2e.tgtMuonStops
       physics: {
 	pdgId : 2212
+      	genId : muonCapture
 	spectrumShape: flat
 	spectrumVariable: kineticEnergy
```

Comment:
I did not catch this one before.   What is the use case for a sample of ejected protons flat in kinetic energy?   We already have a realistic capture spectrum in a different fcl file.

Reply:
This is a historical file, it may not be useful going forwards but it's not a big burden to provide/maintain it either.  'proton.fcl' provides the natural spectrum (model).